workflow-service/src/main/java/com/redhat/parodos/workflow/execution/controller/PubliclyVisible.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.parodos.workflow.execution.controller;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ ElementType.TYPE_USE, ElementType.PARAMETER, ElementType.FIELD })
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = PublicVisibilityValidator.class)
+public @interface PubliclyVisible {
```

Comment:
we have all other Annotation in parodos-model-api.  should we move this to there?

Reply:
I wouldn't move it if it isn't needed for the user/client.
For now, we use it internally for the purpose of validating input within the service only.
Pls let me know if you see a need for it.

Reply:
yes keep it here if it's only to be used for this controller